### PR TITLE
Add support for EdgeQL languages

### PR DESCRIPTION
**Pull request by DAlperin**
Implements https://www.edgedb.com/docs/edgeql/index

## Description
Adds grammars, language definition, and adds test case.

## Checklist:

- [x] **I am adding a new language.**
  - [x] The extension of the new language is used in hundreds of repositories on GitHub.com.
    - Search results for each extension:
		- https://github.com/search?q=extension%3Aedgeql+NOT+nothack&type=Code
		- https://cs.github.com/?scopeName=All+repos&scope=&q=path%3A*.edgeql
    
  - [x] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - https://github.com/edgedb/edgedb-examples
    - Sample license(s):
    - https://github.com/edgedb/edgedb/blob/master/LICENSE
  
  - [x] I have included a syntax highlighting grammar: [https://github-lightshow.herokuapp.com/](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fedgedb%2Fedgedb-editor-plugin%2Fmaster%2Fgrammars%2Fedgeql.cson&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Fedgedb%2Fedgedb-examples%2Fmain%2Fnextjs-todomvc%2Fdbschema%2Fmigrations%2F00001.edgeql&code=)
  - [ ] I have updated the heuristics to distinguish my language from others using the same extension.

### CODE REVIEW

**Comment by DAlperin** [comment]
@lildude this is my first PR to linguist so let me know if I have made any glaring errors :)

**Comment by lildude** [comment]
Looks good from a quick squiz, however usage is still really low for both extensions, even when considering our [temporarily relaxed criteria](https://github.com/github/linguist/issues/5756). Labelling as such.

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -7168,6 +7168,17 @@ eC:
   tm_scope: source.c.ec
   ace_mode: text
   language_id: 413
+edgeql:
```
[the remark this answers is not in the document]
Comment (DAlperin) [reply lib/linguist/languages.yml]: Fixed!

**Comment by DAlperin** [comment]
Fixed the capitalization. As for the usage, fair. It just hit 1.0 yesterday which is how I came across it. Hopefully it will pick up soon.

**Review by lildude — APPROVED** [review]

**Pull request merged**